File: 01.JS-Basic/More Exercises/PB - More Exercises/Nested Conditional Statements/BikeRace.js
function bikeRace(input) {

    let juniors = Number(input[0])
    let seniors = Number(input[1])
    let typeRoute = input[2]

    switch (typeRoute) {
        case "trail":
            console.log(((juniors * 5.5 + seniors * 7) * 0.95).toFixed(2)); break;

        case "cross-country":
            if (juniors + seniors >= 50) {
                console.log((((juniors * 8 + seniors * 9.5) * 0.75) * 0.95).toFixed(2))
            }
            else { console.log(((juniors * 8 + seniors * 9.5) * 0.95).toFixed(2))}; break; 

        case "downhill":
            console.log(((juniors * 12.25 + seniors * 13.75) * 0.95).toFixed(2)); break;
        case "road":
            console.log(((juniors * 20 + seniors * 21.5) * 0.95).toFixed(2)); break;
    }


}
bikeRace(["20", "25", "cross-country"])

File: JAVA-Basic/JavaBasic/src/ConditionalStatementsAdvanced/MoreExercise/BikeRace.java
package ConditionalStatementsAdvanced.MoreExercise;

import java.util.Scanner;

public class BikeRace {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int juniorsCyclists = Integer.parseInt(scan.nextLine());
        int seniorsCyclists = Integer.parseInt(scan.nextLine());
        String typeRoad = scan.nextLine();
        double money = 0.00;

        switch (typeRoad) {
            case "trail":
                money = seniorsCyclists * 7 + juniorsCyclists * 5.5;
                break;
            case "cross-country":
                if (seniorsCyclists + juniorsCyclists >= 50) {
                    money = seniorsCyclists * (9.5 * 0.75) + juniorsCyclists * (8 * 0.75);
                } else {
                    money = seniorsCyclists * 9.5 + juniorsCyclists * 8;
                }
                break;
            case "downhill":
                money = seniorsCyclists * 13.75 + juniorsCyclists * 12.25;
                break;
            case "road":
                money = seniorsCyclists * 21.5 + juniorsCyclists * 20;
                break;
        }
        System.out.printf("%.2f", money * 0.95);
    }
}
